docker-compose.yml

```diff
@@ -8,14 +8,13 @@ services:
       context: .
     environment:
       RUST_LOG: debug
-      RAD_PASSPHRASE: seed
+      RAD_PASSPHRASE: ${RAD_PASSPHRASE:-seed}
       RUST_BACKTRACE: 1
       GIT_TRACE: 1
       GIT_TRACE_PACKET: 1
     volumes:
-      - /mnt/radicle/heartwood:/root/
+      - ${RAD_ROOT:-/mnt/radicle/heartwood}:/root/
```

Comment:
Hmm this should just be `$RAD_HOME` I think?

Reply:
well, I originally thought the same, and this is why I opened #337